examples/client.go

```diff
@@ -107,7 +107,7 @@ func main() {
 		clientOptions = append(clientOptions, realis.SchedulerUrl(url))
 	}
 
-	if clientKey != "" || clientCert != "" || caCertsPath != "" {
+	if clientKey != "" && clientCert != "" && caCertsPath != "" {
```

Comment:
"caCertsPath" and "clientKey + clientCert" are 2 independent sets of parameters, so they should be processed separately